Source/Pragmatic.EntityFramework.Tests.Integration/GetOneQueryHandlerTests.cs

```diff
@@ -9,25 +10,52 @@ namespace Pragmatic.EntityFramework.Tests.Integration
 {
     // ReSharper disable InconsistentNaming
     [TestFixture]
-    public class GetOneQueryHandlerTests // TODO-IG: Define integration tests in a persistence independent way and test them for all persistences.
+    public class GetOneQueryHandlerTests // TODO-IG: Define integration tests in a persistence independent way and test them for all persistence's.
     {
         [Test]
-        public void Returns_person_by_name()
+        public void Returns_person_by_name_ordered_by_age()

```

Comment:
Honestly, I wouldn't bother with it so far. Current tests in Pragmatic can hardly be called tests. Those which are currently in are added after we noticed bugs that were fixed. The components are currently not tested at all, except indirectly through the projects that use Pragmatic. I would say, once we really start covering Pragmatic with proper tests, than we can write on for this case as well :-)